npm-debug.log

```diff
@@ -0,0 +1,27 @@
+0 info it worked if it ends with ok
+1 verbose cli [ '/Users/dave/.nvm/versions/v7.0.0/bin/node',
+1 verbose cli   '/Users/dave/.nvm/versions/v7.0.0/bin/npm',
+1 verbose cli   'run',
+1 verbose cli   'install' ]
+2 info using npm@3.10.8
+3 info using node@v7.0.0
+4 verbose stack Error: missing script: install
+4 verbose stack     at run (/Users/dave/.nvm/versions/v7.0.0/lib/node_modules/npm/lib/run-script.js:151:19)
+4 verbose stack     at /Users/dave/.nvm/versions/v7.0.0/lib/node_modules/npm/lib/run-script.js:61:5
+4 verbose stack     at /Users/dave/.nvm/versions/v7.0.0/lib/node_modules/npm/node_modules/read-package-json/read-json.js:356:5
+4 verbose stack     at checkBinReferences_ (/Users/dave/.nvm/versions/v7.0.0/lib/node_modules/npm/node_modules/read-package-json/read-json.js:320:45)
+4 verbose stack     at final (/Users/dave/.nvm/versions/v7.0.0/lib/node_modules/npm/node_modules/read-package-json/read-json.js:354:3)
+4 verbose stack     at then (/Users/dave/.nvm/versions/v7.0.0/lib/node_modules/npm/node_modules/read-package-json/read-json.js:124:5)
+4 verbose stack     at /Users/dave/.nvm/versions/v7.0.0/lib/node_modules/npm/node_modules/read-package-json/read-json.js:311:12
+4 verbose stack     at /Users/dave/.nvm/versions/v7.0.0/lib/node_modules/npm/node_modules/graceful-fs/graceful-fs.js:78:16
+4 verbose stack     at tryToString (fs.js:425:3)
+4 verbose stack     at FSReqWrap.readFileAfterClose [as oncomplete] (fs.js:412:12)
+5 verbose cwd /Users/dave/dev/V_Is_For_Vince
+6 error Darwin 16.1.0
+7 error argv "/Users/dave/.nvm/versions/v7.0.0/bin/node" "/Users/dave/.nvm/versions/v7.0.0/bin/npm" "run" "install"
+8 error node v7.0.0
+9 error npm  v3.10.8
+10 error missing script: install
+11 error If you need help, you may report this error at:
+11 error     <https://github.com/npm/npm/issues>
+12 verbose exit [ 1, true ]
```

Comment:
This needs to go in the .gitignore

Add `npm-debug.log*` to .gitignore and remove this file before merging